React/tic-tac/src/redux/auth/actions.js

```diff
@@ -0,0 +1,35 @@
+import Service from '@services/UserService';
+
+export const actions = { LOGIN: 'LOGIN', LOGOUT: 'LOGOUT' };
+
+function logInTrue(payload) {
+  return {
+    type: actions.LOGIN,
+    loggedIn: payload
+  };
+}
+
+function logIn(payload) {
+  return dispatch => {
+    Service(payload)
+      .then(response => {
+        if (response.ok) {
+          dispatch(logInTrue(true));
+        } else {
+          throw new Error();
+        }
+      })
+      .catch(() => {
+        // eslint-disable-next-line
+        alert("Error al iniciar sesión");
```

Comment:
Try not to use `alert` to give information to a user. Try showing the message next to the element that has caused the error. 🙂